<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>600</width>
                <height>1024</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QTableWidget" name="tableWidget">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>190</y>
                        <width>579</width>
                        <height>731</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>20</pointsize>
                    </font>
                </property>
                <column>
                    <property name="text">
                        <string>Vezeték</string>
                    </property>
                </column>
                <column>
                    <property name="text">
                        <string>Doboz</string>
                    </property>
                </column>
                <column>
                    <property name="text">
                        <string>Állapot</string>
                    </property>
                </column>
            </widget>
            <widget class="QLabel" name="lblStatus">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>90</y>
                        <width>571</width>
                        <height>91</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>25</pointsize>
                    </font>
                </property>
                <property name="layoutDirection">
                    <enum>LeftToRight</enum>
                </property>
                <property name="styleSheet">
                    <string notr="true">color:#fff;</string>
                </property>
                <property name="text">
                    <string>Várakozás a vezetékre</string>
                </property>
                <property name="alignment">
                    <set>AlignCenter</set>
                </property>
            </widget>
            <widget class="QPushButton" name="btnNewBox">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>930</y>
                        <width>581</width>
                        <height>81</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>20</pointsize>
                    </font>
                </property>
                <property name="styleSheet">
                    <string notr="true">background-color: #FF5722;
                        color: #fff;</string>
                </property>
                <property name="text">
                    <string>Új doboz</string>
                </property>
            </widget>
            <widget class="QLabel" name="lblCount">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>20</y>
                        <width>581</width>
                        <height>61</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>18</pointsize>
                    </font>
                </property>
                <property name="styleSheet">
                    <string notr="true">color: #fff;</string>
                </property>
                <property name="text">
                    <string>TextLabel</string>
                </property>
                <property name="alignment">
                    <set>AlignCenter</set>
                </property>
            </widget>
        </widget>
    </widget>
    <resources />
    <connections />
</ui>